MC/run/PWGGAJE/run_jets_embedding.sh

```diff
@@ -15,7 +15,7 @@
 RNDSEED=${RNDSEED:-0}   # [default = 0] time-based random seed
 NSIGEVENTS=${NSIGEVENTS:-2}
 NBKGEVENTS=${NBKGEVENTS:-1}
-NTIMEFRAMES=${NTIMEFRAMES:-5}
+NTIMEFRAMES=${NTIMEFRAMES:-1}
```

Comment:
do you want to commit this change?

Reply:
It is an example I was testing, I will change it back